### ISSUE TITLE
Religious wonder

### ISSUE THREAD

**Issue by Der-Zorn-Gottes** [LoneGazebo/Community-Patch-DLL#1991]
Bug Report Template

Mod Version (i.e Date - (4/23b)):6/15/1

1. Mod List (if using standard CPP set, leave blank):

2. Type of error (i.e. crash, interface bug, AI quirk): 

Sorry to bring this up once again, but it bacame even weirder.
Initially, it was all right, I needed around 12%, which is expected for this mapsize (huge).
But several turns later, in which I spread my religion to a lot of cities, I get this:

![20160618192211_1](https://cloud.githubusercontent.com/assets/16065630/16173605/1982dc90-35a7-11e6-9dc1-e4c607981dc6.jpg)

It is as if there is some event, maybe era change or I don't know, that set's the correct number for huge map back to a default number (which seems to be the one of a duell map). That's my interpretation, for what it's worth.

3. Steps to reproduce:

4. Additional information:

---------------------------
Supporting information:

Please note that you can attach .zip files to Issues by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files: Database.log and Lua.log needed
2. Minidump file (located in your Civ5.exe directory)
3. Screenshots (if needed)

**Comment by pandasnail** [comment]
You sure its Duel? Its 100% for sure Small for me. And Duel is 50% required, you've got 29% spread? Whereas Small is 30%. Interesting that it was briefly right tho. 

Also, I finally tested it out by switching back and reloading an old save, and it really isn't buildable at the proper number, its not just a weird display thing. (That was last version tho fwiw)

**Comment by LoneGazebo** [comment]
I'm not sure truth be told. There's no interaction with events or anything. Just a basic arithmetic function.

**Comment by Der-Zorn-Gottes** [comment]
Loading this game resulted in the correct value again. 
I have no idea what's going on any more.

(Panda, you were right with small).

**Comment by Der-Zorn-Gottes** [comment]
And like 5 turns later, the required percentage switched from 1% to 18%.

**Comment by LoneGazebo** [comment]
Odd. Are you getting any weird religion change messages in the process?

**Comment by Der-Zorn-Gottes** [comment]
I noticed none.

**Comment by LoneGazebo** [comment]
Did you own more than one holy city?

**Comment by pandasnail** [comment]
Is there any kind of test game I can make that would help with this? Especially since my warring plans did not go well against the new AI (and as silly as it, I really really like the animations), I have time I think to make a quick speed duel map and get it to 30%. But if you or Ilteroi opened the save and it just said 'need 20% more' obvi that would help not at all. But if there is something like that I can do, let me know.

**Comment by LoneGazebo** [comment]
I'm primarily curious to see if ownership of holy cities and/or other religious cities is affecting it somehow.

**Comment by pandasnail** [comment]
Well, the first game I noticed it, I was actually playing the One City Challenge mode. In the second game, I think I edited the SQL before conquering another Holy City, but can't be sure. Since then I've been playing with the changed number in the SQL, so that's all the data I've got.

I could use the 'autoplay' feature in the firetuner to look at what the number for all six religions in my current game, at least when shifted to a human. (I currently have it at the proper number for Huge, I'd have to set Small back to 30% and delete the cache.) Would that be helpful data? At least one is a conquest if that matters. Nobody has two tho I think, there's exactly two on the other continent. Unfortch, it would take me quite awhile to conquer two holy cities against the new AI.
